pkg/api/create_person_handler.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"encoding/json"
+
+	"github.com/darchlabs/api-example/pkg/person"
+	"github.com/go-playground/validator/v10"
+)
+
+type CreatePersonHandlerInputs struct {
+	Name string `validate:"required"`
+	Age  int64  `validate:"required"`
+}
+
+type CreatePersonHandler struct {
+	storage PersonStorage
+}
+
+func NewCreatePersonHandler(ps PersonStorage) *CreatePersonHandler {
+	return &CreatePersonHandler{
+		storage: ps,
+	}
+}
+
+func (cp CreatePersonHandler) Invoke(ctx *handlerCtx) *handlerRes {
+	// Validate json schema data
+	validate := validator.New()
+	validate.Struct(ctx.ps)
```

Comment:
Are you sure that the code is using this validation? 
Did you try each edge case using postman? 

In this case, I think that you must implement a Test because you can try all different input cases.
The same for all handlers